pkg/controller/studyjob/katib_api_util.go

```diff
@@ -73,7 +73,6 @@ func initializeStudy(instance *katibv1alpha1.StudyJob, ns string) error {
 		return err
 	}
 	instance.Status.SuggestionParameterID = sPID
-	instance.Status.SuggestionCount += 1
```

Comment:
why is this removed?

Reply:
SuggestionCount should record how many times the studyJob had called suggestion, and this value should **+= 1** every time after calling suggestion.
Here (initialize studyJob instance and no suggestion called before) instance.Status.SuggestionCount should be 0 (0 by default).

Reply:
@YujiOshima Is this the intended behavior? Should we update the API documentation instead?

Reply:
For now, SuggestionParam["SuggestionCount"] in db change path is [0, 2, 3, ..., requestcount+1].
after this patch its path is [0,1,2,...,requestcount]

Reply:
@hougangliu Thanks.
> SuggestionParam["SuggestionCount"] in db change path is [0, 2, 3, ..., requestcount+1].
This is not intended behavior, we may need to fix. I'm reviewing and testing this PR.

Reply:
LGTM